src/test/scala/scynamo/ScynamoCodecProps.scala

```diff
@@ -92,9 +92,15 @@ class ScynamoCodecProps extends Properties("ScynamoCodec") {
       decodeAfterEncodeIsIdentity(Duration.fromNanos(value): Duration)
     }
 
-  propertyWithSeed("decode.encode === id (year month)", propertySeed) = Prop.forAll { value: YearMonth =>
-    decodeAfterEncodeIsIdentity(value)
-  }
+  propertyWithSeed("decode.encode === id (java duration)", propertySeed) =
+    Prop.forAll(Gen.chooseNum[Long](-9223372036854775807L, 9223372036854775807L)) { value: Long =>
+      decodeAfterEncodeIsIdentity(java.time.Duration.ofNanos(value))
+    }
+
+  propertyWithSeed("decode.encode === id (year month)", propertySeed) = Prop.forAll(decodeAfterEncodeIsIdentity[YearMonth] _)
```

Comment:
I'd prefer this syntax because it's easier to read for beginners but this is just my personal preference.
```suggestion
  propertyWithSeed("decode.encode === id (year month)", propertySeed) = Prop.forAll(decodeAfterEncodeIsIdentity[YearMonth](_))
```